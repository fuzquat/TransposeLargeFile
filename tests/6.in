1a	1b	1c	1d	1e	1f	1g
2a	2b	2c	2d	2e	2f	2g
3a	3b	3c	3d	3e	3f	3g
5a	4b	4c	4d	4e	4f	4g
5a	5b	5c	5d	5e	5f	5g
6a	6b	6c	6d	6e	6f	6g
7a	7b	7c	7d	7e	7f	7g
8a	8b	8c	8d	8e	8f	8g
9a	9b	9c	9d	9e	9f	9g
10a	10b	10c	10d	10e	10f	10g
11a	11b	11c	11d	11e	11f	11g