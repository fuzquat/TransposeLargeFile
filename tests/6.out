0 0 1a
1 0 1b
2 0 1c
3 0 1d
4 0 1e
5 0 1f
6 0 1g
0 1 2a
1 1 2b
2 1 2c
3 1 2d
4 1 2e
5 1 2f
6 1 2g
0 2 3a
1 2 3b
2 2 3c
3 2 3d
4 2 3e
5 2 3f
6 2 3g
0 3 5a
1 3 4b
2 3 4c
3 3 4d
4 3 4e
5 3 4f
6 3 4g
0 4 5a
1 4 5b
2 4 5c
3 4 5d
4 4 5e
5 4 5f
6 4 5g
0 5 6a
1 5 6b
2 5 6c
3 5 6d
4 5 6e
5 5 6f
6 5 6g
0 6 7a
1 6 7b
2 6 7c
3 6 7d
4 6 7e
5 6 7f
6 6 7g
0 7 8a
1 7 8b
2 7 8c
3 7 8d
4 7 8e
5 7 8f
6 7 8g
0 8 9a
1 8 9b
2 8 9c
3 8 9d
4 8 9e
5 8 9f
6 8 9g
0 9 10a
1 9 10b
2 9 10c
3 9 10d
4 9 10e
5 9 10f
6 9 10g
0 10 11a
1 10 11b
2 10 11c
3 10 11d
4 10 11e
5 10 11f
6 10 11g
